frontend/components/core/ReSearchBar.vue

```diff
@@ -26,14 +26,20 @@
       height="14"
       @click="filter = undefined"
     />
-    <ReInput v-model="filter" placeholder="Search datasets" />
+    <ReInput v-model="filter" :placeholder="placeholder" />
   </ReInputContainer>
 </template>
 <script>
 import "assets/icons/search";
 import "assets/icons/cross";
 
 export default {
+  props: {
+    placeholder: {
+      type: String,
+      required: true,
```

Comment:
We can define a generic value for placeholder: `Search`. It's too much require a placeholder value